package-lock.json

```diff
@@ -386,7 +386,7 @@
     "absolute": {
       "version": "0.0.1",
       "resolved": "https://registry.npmjs.org/absolute/-/absolute-0.0.1.tgz",
-      "integrity": "sha512-Zs3Sz3lmEIH76fvnc4jto4CpuI5bhU2TD+jtBYVIfXnjFm+49y1opKxNy4gtNF+xSLBL7hXRpLwypc+o2MzBIQ==",
+      "integrity": "sha1-wigi+H4ck59XmIdQTZwQnEFzgp0=",
```

Comment:
Looks like node.js v12.11.1 downgraded the integrity hashes of certain pre-existing dependencies from SHA512 to SHA1... might be a show-stopper.

If we push this back to next week I could try updating to a newer version and seeing if this fixes itself.